techniques/html/H90.html

```diff
@@ -51,44 +51,41 @@
 
 <div class="default-grid with-gap leftcol">
 <aside class="box nav-hack sidebar standalone-resource__sidebar">
-    <header class="box-h">Page Contents</header>
+    <header class="box-h">このページの内容</header>
     <div class="box-i">
       <nav aria-label="page contents" class="navtoc">
-        <ul><li><a href="#technique">About this Technique</a></li>
-<li><a href="#description">Description</a></li>
-<li><a href="#examples">Examples</a></li>
-<li><a href="#resources">Related Resources</a></li>
-<li><a href="#related">Related Techniques</a></li>
-<li><a href="#tests">Tests</a></li>
+        <ul><li><a href="#technique">このテクニックについて</a></li>
+<li><a href="#description">解説</a></li>
+<li><a href="#examples">事例</a></li>
+<li><a href="#resources">関連リソース</a></li>
+<li><a href="#related">関連テクニック</a></li>
+<li><a href="#tests">検証</a></li>
 </ul>
       </nav>
     </div>
 </aside>
 
 <main id="main" class="standalone-resource__main">
 		
-<h1><span>Technique H90:</span>Indicating required form controls using <code class="language-html">label</code> or <code class="language-html">legend</code></h1>
+<h1><span>テクニック H90:</span><code class="language-html">label</code> 要素又は <code class="language-html">legend</code> 要素を使用して、必須のフォームコントロールを明示する</h1>
 
 
 
 
 <section id="technique" class="box">
-	<h2 class="box-h box-h-icon">About this Technique</h2>
+	<h2 class="box-h box-h-icon">このテクニックについて</h2>
 	<div class="box-i">
 
 
   
   <p>
-    This technique relates to
-    
-<a href="https://w3c.github.io/wcag/understanding/labels-or-instructions">3.3.2: Labels or Instructions</a>
-(<strong>Sufficient</strong>
-  when used with <a href="../general/G131">G131: Providing descriptive labels</a>).</p>
+    このテクニックは、<a href="https://waic.jp/translations/WCAG22/Understanding/labels-or-instructions">3.3.2: ラベル又は説明</a>
+(<a href="../general/G131">G131: Providing descriptive labels</a>と併用する場合に<strong>十分なテクニック</strong>) に関連している。</p>
```

Comment:
「relates to ...」ですが、他のテクニックと合わせて、「に関連している」を「に関連する」にしてみました。
```suggestion
(<a href="../general/G131">G131: Providing descriptive labels</a>と併用する場合に<strong>十分なテクニック</strong>) に関連する。</p>
```